scripts/bump-version.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+TRAVIS_BRANCH=$1
+echo "TRAVIS_BRANCH IS ${TRAVIS_BRANCH}"
+
+PACKAGE_NAME="$(node -pe "require('./package.json')['name']")"
+echo "PACKAGE_NAME is ${PACKAGE_NAME}"
+
+CURRENT_VERSION="$(npm view ${PACKAGE_NAME}@latest version)"
+echo "CURRENT_VERSION is $CURRENT_VERSION"
+
+# If not on master branch ...
+if [ "$TRAVIS_BRANCH" != 'master' ]; then
+
+    # ... obtain and tokenize the latest version according to tag
+    if [ "$TRAVIS_BRANCH" = 'test' ]; then
+        LATEST_TAG_VERSION="$(npm view ${PACKAGE_NAME}@beta version)"
+    elif [ "$TRAVIS_BRANCH" = 'dev' ]; then
+        LATEST_TAG_VERSION="$(npm view ${PACKAGE_NAME}@alpha version)"
+    else
+        LATEST_TAG_VERSION="$(npm view ${PACKAGE_NAME}@canary version)"
+    fi
+
+    echo "LATEST_TAG_VERSION is $LATEST_TAG_VERSION"
+
+    tokens=($(echo $LATEST_TAG_VERSION | tr "-" "\n"))
+    SEMVER=${tokens[0]};
+    PREID=${tokens[1]};
+
+    # If tag version is higher than the 'latest' version in npm
+    # then it means that no new version has been published to master yet and
+    # we shuld continue incrementing the preid based on the tag version
```

Comment:
```bash
    # If the tag version is higher than the "latest" version in npm,
    # then no new stable version has been published from master yet
    # and we should continue incrementing the "preid" based on the tag version 
```

Reply:
Hi @tukusejssirs, I've amended the comments here as per your recommendation.